packages/web/src/scss/components/UNSTABLE_Avatar/_UNSTABLE_Avatar.scss

```diff
@@ -0,0 +1,36 @@
+@use 'theme';
+@use '../../tools/dictionaries';
+
+.UNSTABLE_Avatar {
```

Comment:
As discussed with the PM, the initials may be better not selectable (`user-select: none`).